src/main/java/com/peerrequest/app/model/Entry.java

```diff
@@ -0,0 +1,60 @@
+package com.peerrequest.app.model;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * This class represents an entry.
+ *
+ * @author Ludger Halpick
+ * @author Michael Mildt
+ * @version 0.0.1
+ */
+public class Entry {
```

Comment:
Relation to Category.java and Review.java missing

Reply:
same goes for review relationship